src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
If class ball contains fields number and color. It means that these fields already belong to the class Ball. So field names should be `color` instead `ballColor` and `number` instead `ballNumber`
Also these fields should have modifier `private`.  It's generally better to encapsulate the fields and access them through these methods, as it provides better control and flexibility